tests/test_utils.py

```diff
@@ -0,0 +1,97 @@
+import tempfile
+from pathlib import Path
+from typing import Optional
+
+import pytest
+
+from src.models.models import Config
+from src.utils.utils import ETLUtils
+
+
+@pytest.fixture
+def temp_directories():
+    """
+    Pytest fixture to create temporary input and output directories.
+    """
+    with tempfile.TemporaryDirectory() as input_dir, tempfile.TemporaryDirectory() as output_dir:
+        yield Path(input_dir), Path(output_dir)
+
+
+@pytest.fixture
+def sample_csv_content():
+    """
+    Pytest fixture to provide sample CSV content with a dynamic DATA_DATE.
+    """
+    return r"""FINANCIAL TYPE,SYMBOL,SECURITY NAME,ISIN,PRICE,QUANTITY,REALISED P/L,MARKET VALUE,DATA_DATE
+            Equity,AAPL,Apple Inc.,US0378331005,150.00,10,500.00,1500.00,2023-01-01
+            Equity,GOOGL,Alphabet Inc.,US02079K3059,2800.00,5,14000.00,14000.00,2023-01-01
+            """.strip()
+
+
+def test_to_snake_case():
+    """
+    Test the to_snake_case function with various inputs.
+    """
+    assert ETLUtils.to_snake_case("Applebead") == "applebead"
```

Comment:
This looks like a parametrize too.
Also, it really shows why the class is not giving you anything more than a module and functions would.